### PR TITLE
Replace any occurrence of yum-centos/yum-epel/yum-elrepo with osl-repos equivalents

### PR DESCRIPTION
**Pull request by TheREK3R**

### CODE REVIEW

File: kitchen.dokken.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+platforms:
+  - name: centos-8
+    driver:
+      image: dokken/centos-8
+      pid_one_command: /usr/lib/systemd/systemd
+  - name: centos-7
+    driver:
+      image: dokken/centos-7
+      pid_one_command: /usr/lib/systemd/systemd
```
Comment (ramereth) [inline kitchen.dokken.yml]: Let's not add this yet

Comment (ramereth) [inline kitchen.dokken.yml]: Let's not add this yet

**Review by ramereth — CHANGES_REQUESTED** [review]

File: kitchen.dokken.yml (hunk 1 above)
Comment (TheREK3R) [reply kitchen.dokken.yml]: removed

**Comment by ramereth** [comment]
LGTM

**Review by ramereth — APPROVED** [review]
Please do a minor bump on `~`

**Comment by TheREK3R** [comment]
!bump minor ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a minor-level version bump to v1.3.0.  Have a nice day!

**Pull request merged**